### PR TITLE
Added basic tag items

### PR DESCRIPTION
**Pull request by arthuwu**
All tag items in lines 1-3 have been completed (i.e. everything except the APP selection and route code/gate prediction)

(TagDefinitions.txt for testing only, "Backup" is the old Hong Kong Radar TOPSKY family in case the plugin does a whoopsie)
[TagDefinitions.txt](https://github.com/user-attachments/files/16074670/TagDefinitions.txt)

### CODE REVIEW

File: AT3/AT3Tags.cpp [hunk 1]
```diff
@@ -62,47 +109,221 @@ void AT3Tags::OnGetTagItem(CFlightPlan FlightPlan, CRadarTarget RadarTarget, int
...
+		case TAG_FUNC_APP_SEL_MENU:
+			AT3Tags().OpenPopupList(Area, APPMenuName.c_str(), 1);
+		}
+}
+
 string AT3Tags::GetFormattedAltitude(CFlightPlan& FlightPlan, CRadarTarget& RadarTarget)
 {
-	return string();
+	int altitude = RadarTarget.GetPosition().GetPressureAltitude();
+	int transAlt = AT3Tags().GetTransitionAltitude();
```
Comment (EightSmart) [inline AT3/AT3Tags.cpp:135]: Same as above

File: AT3/AT3Tags.cpp [hunk 2]
```diff
@@ -11,19 +12,50 @@ AT3Tags::AT3Tags() : CPlugIn(EuroScopePlugIn::COMPATIBILITY_CODE, MY_PLUGIN_NAME
...
+			*pRGB = RGB(229, 214, 130);
+			break;
+		case FLIGHT_PLAN_STATE_TRANSFER_FROM_ME_INITIATED:
+			*pRGB = RGB(241, 246, 255);
+			break;
+		case FLIGHT_PLAN_STATE_ASSUMED:
+			*pRGB = RGB(241, 246, 255);
+			break;
+		case FLIGHT_PLAN_STATE_REDUNDANT:
+			*pRGB = RGB(229, 214, 130);
```
Comment (EightSmart) [inline AT3/AT3Tags.cpp:55]: These colors shouldn't be hardcoded in the future, but I think we need to redefine all the colors as member variables anyway so can save for future release

File: AT3/AT3Tags.cpp [hunk 3]
```diff
@@ -62,47 +109,221 @@ void AT3Tags::OnGetTagItem(CFlightPlan FlightPlan, CRadarTarget RadarTarget, int
 	strcpy_s(sItemString, 16, tagOutput.substr(0, 15).c_str());
 }
 
+void OnFunctionCall(int FunctionId, const char* sItemString, POINT Pt, RECT Area) {
+	CFlightPlan FlightPlan = AT3Tags().FlightPlanSelectASEL();
```
Comment (EightSmart) [inline AT3/AT3Tags.cpp:113]: Why do you need to create a new AT3Tags object to call FlightPlanSelectASEL()?

File: AT3/AT3Tags.cpp (hunk 1 above)
Comment (arthuwu) [reply AT3/AT3Tags.cpp:135]: Good question, as you can tell I am not used to OOP in C++

**Pull request merged**